cocos/asset/assets/asset-enum.ts

```diff
@@ -22,10 +22,12 @@
  THE SOFTWARE.
 */
 
-import { Address, Filter as GFXFilter, Format } from '../../gfx';
+import { Address, Filter, Format } from '../../gfx';
```

Comment:
Is there any benefit to change it?

Reply:
For use inline enum functionality. 
Inline enum plugin will not recognize the `as` syntax, so we should not use it as possible.

Reply:
Got it.